src/main/java/com/rarchives/ripme/ripper/rippers/RedditRipper.java

```diff
@@ -45,7 +47,7 @@
 import j2html.tags.ContainerTag;
 import j2html.tags.specialized.DivTag;
 
-public class RedditRipper extends AlbumRipper {
+public class RedditRipper extends  AbstractHTMLRipper{
```

Comment:
I don't think we fully understand the implications of making this change. Let's move any RedditRipper modifications to a new PR.

Reply:
@Thedrogon you still need to remove these changes from this PR.